### PR TITLE
feat: celestia mainnet bootstrappers

### PR DESCRIPTION
**Pull request by smuu**

### CODE REVIEW

File: nodebuilder/p2p/bootstrap.go [hunk 1]
```diff
@@ -37,7 +37,17 @@ func bootstrappersFor(net Network) ([]string, error) {
 
 // NOTE: Every time we add a new long-running network, its bootstrap peers have to be added here.
 var bootstrapList = map[Network][]string{
-	Mainnet: {},
+	Mainnet: {
+		"/dns4/da-bridge-1.celestia-bootstrap.net/tcp/2121/p2p/12D3KooWSqZaLcn5Guypo2mrHr297YPJnV8KMEMXNjs3qAS8msw8",
+		"/dns4/da-bridge-2.celestia-bootstrap.net/tcp/2121/p2p12D3KooWQpuTFELgsUypqp9N4a1rKBccmrmQVY8Em9yhqppTJcXf",
```
Comment (calldata) [inline nodebuilder/p2p/bootstrap.go:42]: Is this a typo? `p2p12D3KooWQpuTFELgsUypqp9N4a1rKBccmrmQVY8Em9yhqppTJcXf` should be `p2p/12D3KooWQpuTFELgsUypqp9N4a1rKBccmrmQVY8Em9yhqppTJcXf`. There's a slash omitted. The same issue occurs in the next lines.

Comment (smuu) [reply nodebuilder/p2p/bootstrap.go]: fixed

Comment (renaynay) [reply nodebuilder/p2p/bootstrap.go]: oh wow good catch

**Review by renaynay — APPROVED** [review]

**Review by vgonkivs — APPROVED** [review]

**Review by walldiss — APPROVED** [review]

**Comment by Wondertan** [comment]
Thinking out loud. Naming of bootstrappers with the node type in dns will backfire. Because node types will change, while DNSes should not be changed. Particularly bridge will disappear as well as we may migrate of full nodes in favour of partial reserved ones for bootstrapping.

cc @celestiaorg/devops

**Review by Wondertan — APPROVED** [review]

**Pull request merged**